server/filestore.go

```diff
@@ -3551,9 +3567,6 @@ func (mb *msgBlock) flushPendingMsgsLocked() (*LostStreamData, error) {
 		}
 	}
 
-	// set write err to any error.
-	mb.werr = err
```

Comment:
This original code would have cleared `mb.werr` if this flush was successful (err would be nil). Is that something we wanted? If so, this change would break that, no?

Reply:
Good catch, will fix.